blog/themes/3mdeb/layouts/authors/list.html

```diff
@@ -1,58 +1,124 @@
+{{ partial "header.html" . }}
 {{ $pageAuthor := .Params.identifier }}
+{{ $paginator := .Paginate (where .Site.Pages "Section" "post").ByPublishDate.Reverse }}
```

Comment:
@artur-rs I tried use pagination code from index.html. At the moment it shows posts wrong order like it's in loop I think here is a problem: "Section" "post"

Reply:
correct paginator: `{{ $paginator := .Paginate (where .Site.Pages "Params.author" $pageAuthor) }}`

Looped articles were caused by second `range` function